app/views/todo_items/_todo_item.html.erb

```diff
@@ -4,8 +4,9 @@
 <%= link_to "Mark as Complete", complete_todo_list_todo_item_path(@todo_list, todo_item.id), method: :patch %>
 -->
 <div class="row clearfix">
+  <% todo_item.assign_status %>
   <% if todo_item.completed? %>
-    <div class="complete">
+    <div class="complete" style="background-color: <%= todo_item.status%>">
```

Comment:
inline style is not recommended anywhere.

Reply:
agreed